gamemodes/terrortown/gamemode/client/cl_radio.lua

```diff
@@ -395,3 +404,14 @@ function GM:PlayerSentRadioCommand(ply, name, target)
 
 	ply:AnimPerformGesture(act)
 end
+
+---
+-- Override or hook in plugin for spam prevention and whatnot. Return true
+-- to block a command.
+-- @param string cmd
+-- @return[default=nil] boolean
+-- @hook
+-- @realm client
+function GM:TTT2ClientRadioCommand(cmd)
```

Comment:
shouldn't this be called `TTT2BlockClientRadioCommand` to better describe the functionality and usage?

Reply:
Then it wouldn't match with it's equivalent on serverside: https://github.com/TTT-2/TTT2/pull/514/files#diff-e34aa4dba182deadbffaaf397db054f7R457